### DISCUSSION TITLE
Submission processing error: BadZipFile

### DISCUSSION THREAD

**Discussion by JonOttesen** [facebookresearch/fastMRI#258] [Q&A] [answered]
Hi,
I have been trying to submit my predictions, but I always end up with a _BadZipFile: File is not a zip file_ error. I have tried uploading with both a .zip or .tar.gz file, but neither file format works.

**Comment by mmuckley** [comment, marked as answer]
Hello @JonOttesen, it could very well depend on which platform you use for the submissions. For sure Amazon S3 and Dropbox are working, but Google Drive does not work right now as documented in Issue #223. Could you try S3 or Dropbox?
